### ISSUE TITLE
Non-linear Navigation 

### ISSUE THREAD

**Issue by jbonlinea** [henrikingo/impress.js#17]
Hi,

Not so much a issue than a question.
I understood the plugin GoTo enable to set up a non linear navigation, great.

However I would like to navigate in a way that seems undoable without coding :
for instance : 1 2 3 4 5 3 6 7 
if I use the plugin GoTo I'll end up in a loop with 3 4 5.

Have I missed anything ?

Thank's for your help

**Comment by henrikingo** [comment]
You're right, it is not possible to define a sequence as you're suggesting here.

It's not difficult to code it though. On `impress:stepleave` event, you could set the `data-goto-next=""` attribute to point to 6 instead of 4. (as a simple example)

However, goto plugin allows you to define different keys to go to different next slide. So you could have RightArrow mapped to 4 and DownArrow mapped to 6, as an example. (See documentation for exact attribute names.)

**Comment by jbonlinea** [comment]
Hi again :-)

I really like the smart use of goto you're suggesting in order to create non linear navigation but it still requires a keyboard while I would idealy look for a solutions usable only with a left-right arrows as you might find in some conferences room !

Great stuff anyway !
thank's

**Comment by henrikingo** [comment]
Your proposal to be able to specify a sequence (other than what comes from reading html top to bottom) makes perfect sense. I was merely trying to describe what is the closest thing that is possible today.
